### ISSUE TITLE
508-defect-2 Fast Follow! [DOCUMENTS]: PDFs MUST be accessibility compliant

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12918]
# [508-defect-2 Fast Follow!](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The generated PDF **must** pass an accessibility check. Before the 10-10CG MVP rolls out to a larger audience, it would help to ensure the base 10-10CG PDF is compliant, so that there isn't a backlog of inaccessible application PDFs.

### Status

Someone in the VA 508 Office is working with the business owner that owns the 10-10CG form PDF to ensure it is accessible. 

It may be helpful to check with the business owner to determine when they will provide a compliant PDF. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a 10-10CG application processor or Caregiver Support Coordinator that uses assistive technology, I want to review the application PDF and understand the content hierarchy and content so that I can do my job processing the application.

## Environment

* Operating System: all
* Browser: all
* Server destination: staging & production

## Steps to Recreate

1. Enter https://staging.va.gov/family-member-benefits/apply-for-caregiver-assistance-form-10-10cg/ in the browser
1. Complete the application
1. Once the error appears that there is a problem and the application cannot be submitted, download the PDF
1. Using Adobe Acrobat Pro, run an Accessibility Check on the PDF
1. Verify that the errors shown in the screenshot appear

## Possible Fixes (optional)

Remediate the base PDF.

## WCAG or Vendor Guidance (optional)

* [PDF Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/pdf)

* [Adobe PDF file format accessibility features combined with Adobe® Acrobat® and Adobe Reader® allow universal access to documents](https://www.adobe.com/accessibility/pdf/pdf-accessibility-overview.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-26 at 2 32 30 PM](https://user-images.githubusercontent.com/57469/91342543-37360d80-e7a9-11ea-8ce0-bec6c011855d.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by ala-yna** [comment]
Closing this - we now use a 508-compliant base PDF!
